scripts/screener/screener.states.ts

```diff
@@ -7,42 +7,44 @@ import path from 'path';
 import getScreenerSteps from './screener.steps';
 import { ScreenerState } from './screener.types';
 
-const baseUrl = `${process.env.DEPLOYURL}/react-northstar-screener`;
-const examplePaths = glob.sync('packages/fluentui/docs/src/examples/**/*.tsx', {
-  ignore: ['**/index.tsx', '**/*.knobs.tsx', '**/BestPractices/*.tsx', '**/Playground.tsx'],
-});
-
-const pathFilter = process.env.SCREENER_FILTER;
-const filteredPaths: string[] = minimatch.match(examplePaths, pathFilter || '*', {
-  matchBase: true,
-});
-
-if (pathFilter) {
-  console.log(chalk.bgGreen.black(' --filter '), pathFilter);
-  filteredPaths.forEach(filteredPath => console.log(`${_.repeat(' ', 10)} ${filteredPath}`));
-}
-
-const getStateForPath = (examplePath: string): ScreenerState => {
-  const { name: exampleNameWithoutExtension, base: exampleNameWithExtension, dir: exampleDir } = path.parse(
-    examplePath,
-  );
-
-  const rtl = exampleNameWithExtension.endsWith('.rtl.tsx');
-  const exampleUrl = _.kebabCase(exampleNameWithoutExtension);
-  const pageUrl = `${baseUrl}/maximize/${exampleUrl}/${rtl}`;
-
-  return {
-    url: pageUrl,
-    name: exampleNameWithExtension,
-
-    // https://www.npmjs.com/package/screener-runner#testing-interactions
-    steps: getScreenerSteps(pageUrl, `${exampleDir}/${exampleNameWithoutExtension}.steps`),
+export default function getScreenerStates() {
```

Comment:
This file used to require all of northstar, refactored this to be a factory to only require northstar when needed

Reply:
the logic for particular library should not live here as it introduces tight coupling. I understand this PR purpose is to quick fix current behaviours, but would be great if we can come up with long term solution.